app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
the nifty yield [here](https://github.com/ManageIQ/manageiq/pull/14022/files#diff-1588bc23945c2827748c43d7c14a3b0cR31) let's you do 

```ruby
has_inventory cloud_manager.vms do |collection, persister|
  collection.parent = persister.manager.parent_manager
  collection.dependency_attributes[: orchestration_stacks] = persister.orchestration_stacks
end
```

but hopefully this would mostly be done in the core `ManagerRefresh::Inventory::AutomationManager` so provider developers dont have to deal with it.